UPGRADING.md

```diff
@@ -98,7 +99,7 @@ that we plan to make to this set of commands and options.
   hardly useful since most users will only use a subset of all the binstubs
   available to them. Also, it would force the introduction of a bunch of most
   likely unused files into source control. Because of this, binstubs now must
-  must be created and checked into version control individually.
+  be created and checked into version control individually.
 
 * The `bundle config` command has a new subcommand-based interface.
```

Comment:
Since we are here, could you remove this paragraph, and also the one about `bundle update --all` since we ended up postponing those deprecations?

Reply:
If they are completely removed, readers who read this document before might be confused. So I marked them "postponed".